src/components/RichTextEditor/TextStyleDropdown/ListBox.tsx

```diff
@@ -0,0 +1,29 @@
+/* eslint-disable @typescript-eslint/no-use-before-define */
+import { AriaListBoxOptions, useListBox } from "@react-aria/listbox";
+import { ListState } from "@react-stately/list";
+import React, { FC, RefObject, useRef } from "react";
+import { ListBoxSection } from "./ListBoxSection";
+import { Option } from "./Option";
+
+interface ListBoxProps extends AriaListBoxOptions<unknown> {
+    listBoxRef?: RefObject<HTMLUListElement>;
+    state: ListState<unknown>;
+}
+
+export const ListBox: FC<ListBoxProps> = (props) => {
+    const ref = useRef<HTMLUListElement>(null);
```

Comment:
Not sure if it makes a difference, but I'd add `null` to make the typing more specific (same for all the other useRefs)
```suggestion
    const ref = useRef<HTMLUListElement | null>(null);
```

Reply:
Component not used anymore / deleted.